src/ndn/transport/face.py

```diff
@@ -100,3 +101,65 @@ def __init__(self, host: str = '', port: int = 0):
     async def open(self):
         self.reader, self.writer = await aio.open_connection(self.host, self.port)
         self.running = True
+
```

Comment:
Keep StreamFace and its subclasses in `stream_face.py` for backwards compatibility.
Put UdpFace in a new file `udp_face.py`.
The Face type should be in `face.py`, but re-exported through `stream_face.py` for backwards compatibility.